cognite/client/data_classes/transformation_jobs.py

```diff
@@ -0,0 +1,249 @@
+import asyncio
+import time
+from enum import Enum
+from uuid import UUID
+
+from cognite.client.data_classes._base import *
+from cognite.client.data_classes.transformations_common import RawTable, TransformationDestination
+
+
+class TransformationJobStatus(str, Enum):
+    RUNNING = "Running"
+    CREATED = "Created"
+    COMPLETED = "Completed"
+    FAILED = "Failed"
+
+
+class TransformationJobMetric(CogniteResource):
+    """The transformation job metric resource allows following details of execution of a transformation run.
+
+    Args:
+        id (int): A server-generated ID for the object.
+        timestamp (int): Time of the last metric update.
+        name (str): Name of the metric.
+        count (int): Value of the metric.
+        cognite_client (CogniteClient): The client to associate with this object.
+    """
+
+    def __init__(self, id: int = None, timestamp: int = None, name: str = None, count: int = None, cognite_client=None):
+        self.id = id
+        self.timestamp = timestamp
+        self.name = name
+        self.count = count
+        self._cognite_client = cognite_client
+
+    @classmethod
+    def _load(cls, resource: Union[Dict, str], cognite_client=None):
+        instance = super(TransformationJobMetric, cls)._load(resource, cognite_client)
+        return instance
+
+
+class TransformationJobMetricList(CogniteResourceList):
+    _RESOURCE = TransformationJobMetric
+    _ASSERT_CLASSES = False
+
+
+class TransformationJob(CogniteResource):
+    """The transformation job resource allows following the status of execution of a transformation run.
+
+    Args:
+        id (int): A server-generated ID for the object.
+        uuid (UUID): A server-generated UUID for the object.
+        status (TransformationJobStatus): Status of the job.
+        transformation_id (int): Server-generated ID of the transformation.
```

Comment:
and this?

Reply:
this is introduced to standardize the API, aiming to replace uuid.